src/main/java/tests/ShopServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package tests;
+
+import core.basesyntax.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ShopServiceTest {
+
+    private ShopService shopService;
```

Comment:
It's a good practice to initialize the objects in the `setUp` method with the `@BeforeEach` annotation, but ensure that each test is independent and does not affect the others. Currently, the `transactions` list is shared among tests which can lead to side effects if the list is modified in one test and those changes carry over to another test.